index.html

```diff
@@ -6,13 +6,163 @@
+
+        <section class="program pb-12 w-full" id="program">
+            <h2 class="text-center text-3xl program-title pt-16">Main Program</h2>
+            <div class="line"></div>
+            <div class="mt-12 grid md:grid-cols-4 gap-y-4 md:gap-x-2 mx-3 md:mx-10">
+                <div class="program-card gap-x-2 flex py-8 px-2 md:flex-col items-center justify-center">
+                    <i class="fa-solid fa-umbrella-beach "></i>
+                    <h2 class="text-lg md:text-xl program-subtitle pt-2">Entertainment</h2>
+                    <p class="program-desc pt-2 font-light text-center text-sm md:text-base">you will get the chance to
+                        see and meet your
+                        favorite stars
+                        from
+                        Pakistan
+                        showbiz industry</p>
+                </div>
+                <div class="program-card gap-x-2  flex py-8 px-2 md:flex-col items-center justify-center">
+                    <i class="fa-solid fa-award"></i>
+                    <h2 class="text-lg md:text-xl program-subtitle pt-2">Awards</h2>
+                    <p class="program-desc pt-2 font-light text-center text-sm md:text-base">Awards would be given to
+                        best candidates in their given category in tv dramas and movies.</p>
+                </div>
+                <div class="program-card gap-x-2 flex py-8 px-2 md:flex-col items-center justify-center">
+                    <i class="fa-solid fa-umbrella-beach "></i>
+                    <h2 class="text-lg md:text-xl program-subtitle pt-2">Recreation</h2>
+                    <p class="program-desc pt-2 font-light text-center text-sm md:text-base">Snacks would be distributed
+                        among audience and dinner would be served to them. kids are welcomed</p>
+                </div>
+                <div class="program-card gap-x-2  flex py-8 px-2 md:flex-col items-center justify-center">
+                    <i class="fa-solid fa-award"></i>
+                    <h2 class="text-lg md:text-xl program-subtitle pt-2">Entertainment</h2>
+                    <p class="program-desc pt-2 font-light text-sm md:text-base text-center">You will be
+                        eager to take autograph of your favorite actor/actress, so there is a meet and greet
+                        session</p>
+                </div>
+            </div>
+            <h3 class="text-center pt-8 text-2xl"><a href="#" class="underline  program-link">View full
+                    Program</a>
+            </h3>
+        </section>
+
+        <!-- speakers section -->
+
+        <section class="py-12 w-full speakers" id="speakers">
+            <h2 class="text-center text-3xl speakers-title pt-8">Featured Speakers</h2>
+            <div class="line"></div>
+            <div class="mt-12 grid md:grid-cols-2 gap-y-6 md:gap-x-4 mx-3 md:mx-10" id="speaker-container">
+            </div>
+        </section>
```

Comment:
1. - [ ] Great Job Implementing this section dynamically, I know you can add an unlimited number of cards in this sections thanks to your great js script, but let's make them 6 cards shown by default to match the design currently only 4 shown

![image](https://user-images.githubusercontent.com/89282221/190476893-0a3b0f45-9162-43d0-9fcd-cd7b86886357.png)

---------------------------------

2. kindly try to follow the Cindy design as much as possible, currently you have those styling issues that need to be fixed : 

- [ ]  kindly reduce the size of the images they are so big compared to the design, and try to add the missing background image/patterns behind the profile image for each one of the cards
- [ ]  kindly update the color of the sub-title/job section of the card to `#EC5242`
- [ ]  kindly add the divider between the job title and the description as shown in the design




Your Design         | Cindy's Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/190477448-ca5b89f0-4589-40a0-9e67-766ce3c55e7c.png) | ![image](https://user-images.githubusercontent.com/89282221/190478827-cd2f506d-22a8-4f0e-baa1-9b9d2cef85b2.png)